app/src/main/AndroidManifest.xml

```diff
@@ -150,17 +142,6 @@
         <category android:name="android.intent.category.DEFAULT" />
       </intent-filter>
     </activity>
-    <activity
-      android:name=".zim_manager.ZimManageActivity"
-      android:label="@string/choose_file"
-      android:launchMode="singleTop">
-      <intent-filter>
-        <action android:name="org.kiwix.kiwixmobile.utils.KiwixSearchWidget.TEXT_CLICKED" />
```

Comment:
this should mean that the widget is now broken? Hmm actually these filters make no sense on this activity in the first place, I think they should be on Main

Reply:
I don't know what they are used for. Like this? 

```
<activity
      android:name=".main.KiwixMainActivity"
      android:configChanges="orientation|keyboardHidden|screenSize"
      android:label="@string/app_name"
      android:launchMode="singleTop"
      android:windowSoftInputMode="adjustPan">
      ....
      <intent-filter>
        <action android:name="android.appwidget.action.APPWIDGET_UPDATE" />
        <action android:name="KiwixSearchWidget.TEXT_CLICKED" />
        <action android:name="KiwixSearchWidget.ICON_CLICKED" />
        <action android:name="KiwixSearchWidget.MIC_CLICKED" />
        <action android:name="KiwixSearchWidget.STAR_CLICKED" />
      </intent-filter>
```

Reply:
Yeah, looks about right, you could test the widget as well but I am not entirely sure it is working right now or ever really worked well

Reply:
If it ever worked, it's not working now. It's not even showing up as an option for widgets.

Reply:
nvm, it shows up now but does not work